documentation/milestones/individual_milestones/search_mvp.md

```diff
@@ -0,0 +1,218 @@
+- **[User Research Incentives](https://github.com/HHS/grants-equity/issues/84):** Enables us to engage stakeholders for user testing of the new search functionality
+
+Is there any notable functionality we do ***not*** expect to be in place before works starts on this deliverable?
+
+- **AuthN/AuthZ:** Authentication (AuthN) and Authorization (AuthZ) will be the focus of a subsequent deliverable, so the MVP is not expected to support any features that require users to be logged in.
+
+### Open questions
+<!-- Optional -->
+
+#### Does the metadata and the text of the opportunity need to be translated?
+
+TODO
+
+#### What metadata is currently available about each opportunity?
+
+TODO
+
+### Not doing
+<!-- Optional -->
+
+The following work will *not* be completed as part of this deliverable:
+
+- **Personalization:** Because the MVP will not support a login experience, any personalized features that require authentication (e.g. saving searches) are explicitly descoped from this deliverable.
+- **Full Text Search:** The MVP is only expected to include (a subset of) filtering based on opportunity metadata that is already available in the existing grants.gov database, so searching the full text of the opportunity is explicitly descoped from this deliverable.
+- **Full Text Listing:** Similarly, the first iteration of the opportunity listing page delivered is only expected to include opportunity fields that are already available in the existing grants.gov database. As a result, displaying the full text of the opportunity within the listing page is explicitly descoped from this deliverable.
+
+## Integrations
+
+### Translations
+<!-- Required -->
+
+*Does this milestone involve delivering any content that needs translation?*
+
+TODO
+
+*If so, when will English-language content be locked? Then when will translation be started and completed?*
+
+TODO
+
+### Services going into PROD for the first time
+<!-- Required -->
+
+This can include services going into PROD behind a feature flag that is not turned on.
+
+1. **Search API:** Search functionality will be exposed via the API for the first time in this deliverable
+2. **Search UI:** Search functionality will be exposed via the user interface for the first time in this deliverable
+
+### Services being integrated in PROD for the first time
+<!-- Required -->
+
+Are there multiple services that are being connected for the first time in PROD?
+
+1. **API + Front-end:** This deliverable represents the first time the simpler.grants.gov front-end and API will be integrated in production
+
+### Data being shared publicly for the first time
+<!-- Required -->
+
+Are there any fields being shared publicly that have never been shared in PROD before?
+
+1. TODO
```

Comment:
The data that is being shared should be the same or similar to what exists on grants.gov. We can determine this in the deliverable and through the security review process.

Reply:
I think this would be a perfect callout! We might also want to highlight if we anticipate exposing new fields that are currently available on grants.gov cloud but were not released with the `GET /opportunities` endpoint. 

We don't necessarily need to name the fields directly just highlight that there will be net new opportunity fields relative to our previous deliverable (if that's the expectation)